lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -250,13 +251,18 @@ def test_staff_level(self):
         # update_forum_role(self.course.id, staff_member, FORUM_ROLE_ADMINISTRATOR, 'allow')
 
         for endpoint, args in self.staff_level_endpoints:
+            expected_status = 200
+
+            if endpoint in ['calculate_grades_csv', 'get_ora2_responses']:
+                expected_status = 202
```

Comment:
This line is reported as uncovered. It's a bit concerning when test lines don't get hit. Can you double-check this? https://build.testeng.edx.org/job/edx-platform-report-pr/380/Diff_Coverage_Report/